spec/features/subdomain_handling_spec.rb

```diff
@@ -114,12 +114,12 @@
     context "with apex domain" do
       scenario "visiting root should redirect to home community root" do
         visit "/"
-        expect(current_url).to have_subdomain_and_path("foo", "/")
+        expect(current_url).to have_subdomain_and_path("foo", "/users")
       end
 
       scenario "visiting URL with query string should also redirect" do
         visit "/?bar=123"
-        expect(current_url).to have_subdomain_and_path("foo", "/?bar=123")
```

Comment:
These look perfect! The only thing is this one doesn't quite test anything anymore due to the double redirect.

With your changes, nobody should be expecting /?bar=123 to take them to /meals?bar=123. They can use /meals?bar=123 directly. Which I like better anyway.

So if you change the line above to `visit "/meals?bar=123"` and then expect that the query string is preserved after the redirect, you'll have a good test. I don't think it will past as of now. You may need to do some work to pass the query string along with the redirect. Google will help you I think. Something like `request.query` maybe.